app/frontend/wallet/shelley/transaction/utils.ts

```diff
@@ -1,53 +1,35 @@
 import BigNumber from 'bignumber.js'
-import distinct from '../../../helpers/distinct'
 import {Address, CertificateType, Lovelace, TokenBundle} from '../../../types'
-import {aggregateTokenBundles} from '../../helpers/tokenFormater'
+import {encodeCbor} from '../../helpers/cbor'
 import {TxCertificate, TxPlanAuxiliaryData, TxInput, TxOutput, TxWithdrawal} from '../../types'
-import {MIN_UTXO_VALUE} from './constants'
+import {cborizeSingleTxOutput} from '../shelley-transaction'
 import {estimateTxSize} from './estimateTxSize'
 
-export const computeMinUTxOLovelaceAmount = (tokenBundle: TokenBundle): Lovelace => {
-  // based on https://github.com/input-output-hk/cardano-ledger-specs/blob/master/doc/explanations/min-utxo.rst
-  const quot = (x: number, y: number) => Math.floor(x / y)
-  const roundupBytesToWords = (x: number) => quot(x + 7, 8)
-  // TODO: this to network config or constants
-  const minUTxOValue = new BigNumber(MIN_UTXO_VALUE)
-  // NOTE: should be 2, but a bug in Haskell set this to 0
-  const coinSize = 0
-  const txOutLenNoVal = 14
-  const txInLen = 7
-  const utxoEntrySizeWithoutVal = 6 + txOutLenNoVal + txInLen // 27
+export const computeMinUTxOLovelaceAmount = (
+  address: Address,
+  tokenBundle: TokenBundle
+): Lovelace => {
+  // Reference:
+  // - https://cips.cardano.org/cips/cip55/#thenewminimumlovelacecalculation
+  // - https://hydra.iohk.io/build/15339994/download/1/babbage-changes.pdf page 9
+  const overhead = 160
 
-  // NOTE: should be 29 but a bug in Haskell set this to 27
-  const adaOnlyUtxoSize = utxoEntrySizeWithoutVal + coinSize
+  // taken from https://cexplorer.io/params
+  const coinsPerUTxOByte = 4310
```

Comment:
I'd perhaps rather declare it as:
```
const coinsPerUtxoWord = 34482
const coinsPerUtxoByte = coinstPerUtxoWord / 8
```
Since the param currently doesn't really exist (the explorer is showing it as `coins_per_utxo_size` and perhaps I'd rather reference [this](https://book.world.dev.cardano.org/environments/mainnet/alonzo-genesis.json) as a source, what do you think?